tests/system/providers/cohere/example_cohere_embedding_operator.py

```diff
@@ -0,0 +1,49 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from datetime import datetime
+
+from airflow import DAG
+from airflow.decorators import task
+from airflow.providers.cohere.operators.embedding import CohereEmbeddingOperator
+
+with DAG("example_cohere_embedding", schedule=None, start_date=datetime(2023, 1, 1), catchup=False) as dag:
+    # [START howto_operator_cohere_embedding]
+    texts = [
+        "On Kernel-Target Alignment. We describe a family of global optimization procedures",
+        " that automatically decompose optimization problems into smaller loosely coupled",
+        " problems, then combine the solutions of these with message passing algorithms.",
+    ]
+
+    @task
+    def task_to_get_text():
+        return texts
+
+    def get_task():
+        return texts
+
+    CohereEmbeddingOperator(input_text=texts, task_id="embedding_via_text")
+    CohereEmbeddingOperator(input_text=task_to_get_text(), task_id="embedding_via_task")
+    CohereEmbeddingOperator(input_text=get_task(), task_id="embedding_via_callable")
```

Comment:
I don't think there is much value in having this example - this isn't any different than the the first operator call here, as during parsing this will be resolved before being passed to the operator. (It'd be different if `input_text` could be a callable, e.g. `get_task` vs `get_task()`.)

Reply:
Sure, I agree, removed the later two calls.